elements/pfe-tabs/src/pfe-tab-panel.scss

```diff
@@ -46,29 +64,32 @@ $LOCAL-VARIABLES: (
 /// HORIZONTAL
 /// ===========================================================================
 :host([pfe-variant="earth"]) {
-  --pfe-tabs__panel--Padding: calc(#{pfe-var(container-spacer)} * 2);
-
-  --pfe-tabs__panel--BorderRight: #{pfe-var(ui--border-width)} #{pfe-var(ui--border-style)} #{pfe-var(surface--border)};
-  --pfe-tabs__panel--BorderBottom: #{pfe-var(ui--border-width)} #{pfe-var(ui--border-style)} #{pfe-var(surface--border)};
-  --pfe-tabs__panel--BorderLeft: #{pfe-var(ui--border-width)} #{pfe-var(ui--border-style)} #{pfe-var(surface--border)};
-
 
   @include browser-query(ie11) {
```

Comment:
Since earth is the default styles, the IE11 fallbacks should be defined above, maybe around line 55 with the other settings?